package exceptions.exercise2;



public class Main {
    public static void main(String[] args)throws Exception {
        ArrayOfIntegers array = new ArrayOfIntegers();
        System.out.println(array.sumOfArrayValues());
    }
}
